spp_003_nl toev.: random puntjes special procedure veldwerkpraatje (#151)
inbo/protocolsource: source/spp/mnm/spp_003_nl_peilbuisinst-mnm03.3/04_werkwijze.Rmd

-   als je afwijkt van het beoogde punt, dit te beperken tot 10 m van het beoogde punt;
-   als dit niet mogelijk is, de locatie te beoordelen als niet geschikte steekproefeenheid en dan worden de peilbuis en piëzometer niet geplaatst.

Indien er een kaartlaag beschikbaar is met toevallige ("random") alternatieve plaatsingspuntjes binnen de cel en binnen het doelpolygoon (celkartering), dan is de volgorde van de benummering (stijgend) bepalend voor de keuze van een alternatieve plaatsingspunt.
Kies dan bij voorkeur het celmidden (puntje 0), indien onbeschikbaar puntje 1, enz.

> De alternatieve plaatsingspuntjes zijn naar een bepaald schema toevallig gekozen, maar met een hogere wichting van punten dichter bij ons doelpunt, het celmidden.

Reviewer: ```suggestion
De alternatieve plaatsingspuntjes zijn toevallig gekozen, maar met een grotere kans voor punten dichter bij het celmidden.
```

> Per uitzondering, bv. als onmiddelijk na een ad-hoc-loceval nog geen toevalspuntjes beschikbaar zijn of bij gebrek aan ontvangst, kan deze verdelingsfunctie experimenteel nagebotst worden. 

Reviewer: ```suggestion
Per uitzondering, bv. als onmiddellijk na een ad-hoc-loceval nog geen toevalspuntjes beschikbaar zijn of bij gebrek aan ontvangst van de precisie-GPS-ontvanger, én na overleg met de meetnetverantwoordelijke kan deze verdelingsfunctie experimenteel nagebootst worden.
Tracht steeds de random gegenereerde toevalspuntjes te verkrijgen, eventueel door contact op te nemen met de data scientist die deze op vraag kan genereren.
Gebruik deze aanpak enkel in nood.
```

> Hiervoor dient een redelijk convex voorwerp met een lage elasticiteitsmodulus verticaal naar boven gegooit te worden.

Reviewer: ```suggestion
Hiervoor kunnen randomgetallen gegenereerd worden met een app op je telefoon, tussen de minimale en maximale coördinaten van de easting en de northing.
Als allerlaatste toevluchtsoord kan een bij voorkeur convex voorwerp met een lage elasticiteitsmodulus verticaal naar boven gegooid worden vanuit het celmidden.
```

> Hiervoor dient een redelijk convex voorwerp met een lage elasticiteitsmodulus verticaal naar boven gegooit te worden.

Reviewer: On second thought: the throwing procedure is not statistically sound and I think it is therfore better to remove it and restrict it to the random number generator.

> De windrichting en -sterkte is te compenseren; met de celstructuur is rekening te houden (bv. celkartering, MHQ-proefvlak).

Reviewer: ```suggestion
Tracht hierbij rekening te houden met de windrichting en -sterkte.
```

> Om blessures bij deze procedure te voorkomen mag dit voorwerp niet te zwaar zijn; best geschikt is een WC-rol in een plastieken zakje.

Reviewer: ```suggestion
Om blessures bij deze procedure te voorkomen mag dit voorwerp niet te zwaar zijn.
```

Author: Some context: this whole PR was about ex post rationalization of the special procedure, which we applied on 3/2... I posted this just shortly after our visit to Kalmthout. In retrospective, I really think we had the ideal "tool" for the "job".

We should double-check with @florisvdh whether we really wish to operate this "special procedure" as an exceptional alternative for determining a random point.
(Personally, I am fine with it.)

Reviewer: Double-checked and we agreed on removing the throwing procedure and suggest the use of a random number generator in this exceptional case.

> De punt waar het voorwerp neervalt is de plaatsingspunt en mag niet verder verschuiven.

Reviewer: ```suggestion
De plek waar het voorwerp neervalt of waar je terechtkomt na toepassen van het random gegenereerde nummer is de plaats van installatie en mag niet verder verschuiven.
De procedure wordt herhaald als de zo verkregen installatieplaats buiten de celkarteringspolygoon valt of binnen het MHQ-proefvlak.
```

> Vraag voor deze uitzonderlijke en ingewikkelde procedure vooraf toestemming van en biotieker, een datawetenschapper, een statisticus, en de meetnetverantwoordelijke.

Reviewer: ```suggestion
Vraag voor deze uitzonderlijke en ingewikkelde procedure vooraf toestemming van de meetnetverantwoordelijke.
```

Author: The point was that this procedure is absolutely exceptional, and might ever only be applied once (when all ~~planets~~ colleagues aligned).

Reviewer: Indeed, this procedure can only be applied in exceptional circumstances. If the procedure is approved by a statistician, then approval of its application by the colleague resonsible for this monitoring scheme will suffice. The throwing procedure is not statistically sound and thus should be removed from the protocol, but the use of a random number generator is a sound back-up for rare occasions (preferably none).
